### PR TITLE
Lego.MindstormsEduNXTSoftware version 2.1

### PR DESCRIPTION
**Pull request by CasperNaudts**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/57866)

### CODE REVIEW

File: manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Lego.MindstormsEduNXTSoftware
+PackageVersion: 2.1
+PackageLocale: en-US
+Publisher: Lego
+PackageName: Mindstorms Edu NXT Software
```
Comment (Trenly) [inline manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.locale.en-US.yaml:8]: This must match control panel
```suggestion
PackageName: LEGO MINDSTORMS Edu NXT Software
```

File: manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Lego.MindstormsEduNXTSoftware
+PackageVersion: 2.1
```
Comment (Trenly) [inline manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.locale.en-US.yaml:5]: This must match control panel; When updating this field, you will also need to update the name of the folder the manifests are in
```suggestion
PackageVersion: 2.1.126.0
```

File: manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: Lego.MindstormsEduNXTSoftware
+PackageVersion: 2.1
```
Comment (Trenly) [inline manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.yaml:5]: This must match control panel; When updating this field, you will also need to update the name of the folder the manifests are in
```suggestion
PackageVersion: 2.1.126.0
```

File: manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Lego.MindstormsEduNXTSoftware
+PackageVersion: 2.1
+PackageLocale: en-US
+Publisher: Lego
```
Comment (Trenly) [inline manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.locale.en-US.yaml:7]: Must match control panel
```suggestion
Publisher: LEGO
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Lego.MindstormsEduNXTSoftware
+PackageVersion: 2.1
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://wingettest.z6.web.core.windows.net/NXT_Installer_2.1.0_Global_WIN.exe
```
Comment (lechacon) [inline manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.installer.yaml:9]: This doesn't look like an official URI from the publisher. The link I found was `https://education.lego.com/_/downloads/NXT_Installer_2.1.0_Global_WIN.zip`, which would be blocked until we get .zip support.

Comment (CasperNaudts) [reply manifests/l/Lego/MindstormsEduNXTSoftware/2.1/Lego.MindstormsEduNXTSoftware.installer.yaml:9]: Therefore, I tried repacking the app. Do you have a timeline on implementation of .zip support?

**Pull request closed without merging**